### ISSUE TITLE
body is not a function

### ISSUE THREAD

**Issue by jdmarshall** [tunnckoCore/opensource#133]
#### Support plan

- _which support plan is this issue covered by?_ (e.g. Community, Sponsor, or
  Enterprise):
- _is this issue currently blocking your project?_ (yes/no):
- _is this issue affecting a production system?_ (yes/no):

#### Context

Node@10.19.0 or Node@8.9.4
koa-better-body@3.3.7

#### What are you trying to achieve or the steps to reproduce?

```js
const some = 'properly formatted code example';
```

#### What was the result you got?

```
let body = require('koa-better-body');
let app = new koa();

app.use(body());
```

#### What result did you expect?

3.3.6 has no issues

**Comment by tunnckoCore** [comment]
Yea, sorry about that. I'll fix it in a bit.

For now try `.default`.

```js
let body = require('koa-better-body').default;
let app = new koa();

app.use(body());
```

**Comment by tunnckoCore** [comment]
Might be fixed in the coming 3.3.9 version.
